### ISSUE TITLE
Temporary LU tax rate for 2023

### ISSUE THREAD

**Issue by GregOriol** [kdeldycke/vat-rates#20]
It seems that Luxemburg has decided a temorary change of VAT rates for 2023 as per https://guichet.public.lu/en/actualites/2023/janvier/02-modification-taux-tva.html:

```
A temporary reduction in VAT rates (-1 percentage point) applies from 1 January 2023 to 31 December 2023.

The following [VAT rates](https://guichet.public.lu/en/entreprises/fiscalite/tva/notions/tva.html) are applicable during this period:

standard rate: 16 % (decrease from 17 % to 16 %);
intermediate rate: 13 % (decrease from 14 % to 13 %);
reduced rate: 7 % (decrease from 8 % to 7 %).
The super-reduced rate of 3 % remains unchanged.
```

**Comment by kdeldycke** [comment]
Has been fixed by @timwhitlock with #21.

**Comment by timwhitlock** [comment]
I hope I did it correctly. I just noticed that end dates overlap the next start date in other entries, whereas I used inclusive date ranges. i.e. I did `2023-01-01 -> 2023-12-31`. Should this be `2023-01-01 -> 2024-01-01` ?
